src/main/include/TrajMath.h

```diff
@@ -32,12 +35,20 @@ namespace t34
             double limelight_angle
         );
 
+        void Periodic();
+
+        void PutTelemetry();
+
         double GetFiringAngleDeg();
 
         bool IsInRange();
 
         double GetDistanceFromTarget();
```

Comment:
Sorry, should have thought of this earlier. Because these methods do not alter the state of the object, suggest adding `const`:

```suggestion
        double GetFiringAngleDeg() const;

        bool IsInRange() const;

        double GetDistanceFromTarget() const;
```